packages/v4/src/content/design-guidelines/components/menu/menu.md

```diff
@@ -0,0 +1,82 @@
+---
+id: Menu
+section: components
+---
+
+A menu component that can be used in a variety of contexts whenever the user needs to choose between multiple values, options, or actions. A menu can be opened in a [dropdown](/components/dropdown), a [select list](/components/select), or it can be revealed upon right clicking on a region within a page.
```

Comment:
Change this too... "A **Menu** can be used in a variety...."
It doesn't really tell you what a menu is though, and more so how you can use it. Should we maybe add a sentence like "A **Menu** is a list of items inside a contained that can be used..." or something along those lines

Reply:
OK. I added an additional sentence. See what you think.